incubator/java-openliberty/stack.yaml

```diff
@@ -14,14 +14,14 @@ maintainers:
     email: skurz@us.ibm.com
     github-id: scottkurz
   - name: Adam Wisniewski
-    email: awisniew@us.ibm.com 
+    email: awisniew@us.ibm.com
     github-id: awisniew90
 default-template: default
 requirements:
-    docker-version: ">= 17.09.0"
-    appsody-version: ">= 0.5.0"
+  docker-version: ">= 17.09.0"
+  appsody-version: ">= 0.5.0"
 templating-data:
-    libertyversion: '20.0.0.9'
-    parentpomgroup: 'dev.appsody'
-    parentpomid: 'java-openliberty'
-    parentpomrange: '[0.2, 0.3)'
+  libertyversion: "20.0.0.9"
```

Comment:
I guess going from single->double quotes was sufficiently tested?   I'm just stating my ignorance.   I think it's likely fine and that the tests would probably catch it...just not sure.